libvcs4j-tools/src/test/java/de/unibremen/informatik/st/libvcs4j/spoon/codesmell/missingJavadoc/ArrayListMultimapTest.java

```diff
@@ -0,0 +1,78 @@
+package de.unibremen.informatik.st.libvcs4j.spoon.codesmell.missingJavadoc;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class ArrayListMultimapTest {
+
+    @Test
+    public void create() {
+        final var actual = ArrayListMultimap.create();
+
```

Comment:
Unnecessary new line.

Reply:
Do you have a style guide document declaring those rules?

Reply:
We neither have a strict style guide nor do we use a style checker. It is the responsibility of the developers to use a "sane" style. That being said, I prefer:

1. Tabs for indention, spaces for layout (1 tab = 4 spaces)
2. Maximum width of 80 (exclusive) spaces.
3. Avoid useless new lines.